setup.py

```diff
@@ -1,9 +1,34 @@
\n    def Echo(self, what: 's') -> 's':\n        return what\n\n    @method()\n    def GetVariantDict() -> 'a{sv}':\n        return {\n            'foo': Variant('s', 'bar'),\n            'bat': Variant('x', -55),\n            'a_list': Variant('as', ['hello', 'world'])\n        }\n\n    @dbus_property()\n    def string_prop(self) -> 's':\n        return self._string_prop\n\n    @string_prop.setter\n    def string_prop_setter(self, val: 's'):\n        self._string_prop = val\n\n    @signal()\n    def signal_simple(self) -> 's':\n        return 'hello'\n\nasync def main():\n    bus = await MessageBus().connect()\n    interface = ExampleInterface('test.interface')\n    bus.export('/test/path', interface)\n    # now that we are ready to handle requests, we can request name from D-Bus\n    await bus.request_name('test.name')\n    # wait indefinitely\n    await asyncio.Event().wait()\n\nasyncio.run(main())\n```\n\n## The Low-Level Interface\n\nThe low-level interface works with DBus messages directly.\n\nFor more information, see the [overview for the low-level interface](https://python-dbus-fast.readthedocs.io/en/latest/low-level-interface/index.html).\n\n```python\nfrom dbus_fast.message import Message, MessageType\nfrom dbus_fast.aio import MessageBus\n\nimport asyncio\nimport json\n\n\nasync def main():\n    bus = await MessageBus().connect()\n\n    reply = await bus.call(\n        Message(destination='org.freedesktop.DBus',\n                path='/org/freedesktop/DBus',\n                interface='org.freedesktop.DBus',\n                member='ListNames'))\n\n    if reply.message_type == MessageType.ERROR:\n        raise Exception(reply.body[0])\n\n    print(json.dumps(reply.body[0], indent=2))\n\n\nasyncio.run(main())\n```\n\n## Projects that use python-dbus-fast\n\n- [Bluetooth Adapters](https://github.com/bluetooth-devices/bluetooth-adapters)\n\n## Contributing\n\nContributions are welcome. Development happens on [Github](https://github.com/altdesktop/python-dbus-fast).\n\nBefore you commit, run `pre-commit run --all-files` to run the linter, code formatter, and the test suite.\n\n## Copyright\n\nYou can use this code under an MIT license (see LICENSE).\n\n- © 2019, Tony Crisci\n- © 2022, Bluetooth Devices authors\n\n## Contributors ✨\n\nThanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):\n\n<!-- prettier-ignore-start -->\n<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->\n<!-- markdownlint-disable -->\n<!-- markdownlint-enable -->\n<!-- ALL-CONTRIBUTORS-LIST:END -->\n<!-- prettier-ignore-end -->\n\nThis project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!\n\n## Credits\n\nThis package was created with\n[Cookiecutter](https://github.com/audreyr/cookiecutter) and the\n[browniebroke/cookiecutter-pypackage](https://github.com/browniebroke/cookiecutter-pypackage)\nproject template.\n",
```

Comment:
Does this all really go in there ?

Reply:
It's auto generated by poetry

Reply:
Let me see if I can hack it out without having to maintain it 😉

Reply:
I was actually able to delete it since its not required anymore